pages/tasks/task_details.py

```diff
@@ -11,6 +11,7 @@
 class TaskDetailsPage(Base):
 
     _name_locator = (By.CSS_SELECTOR, '.content-container > h1')
+    _team_locator = (By.CSS_SELECTOR, '.three-fields > a')
```

Comment:
This seems like a pretty fragile locator. Unfortunately there isn't much in the markup to identify the team link. One thing you could do is base the locator on the `href` of the `a` element, indicating that the `href` should include `team`. Are you familiar with how to do that?

Ideally the application would be coded to make this testing easier, by assigning an `id`, for example, to the element. As I am also the developer of the app, I could do that, or, if you were interested in trying to do it yourself I could help you. Is that something you'd be interested in doing?

Also, as this is the locator for the link it might be better to call it `_team_link_locator`.